src/freenas/usr/local/bin/truenas-grub.py

```diff
@@ -21,9 +21,10 @@ def dict_factory(cursor, row):
     c.execute("SELECT * FROM system_advanced")
     advanced = {k.replace("adv_", ""): v for k, v in c.fetchone().items()}
 
+    # We need to allow tpm in grub as sedutil-cli requires it
     config = [
         'GRUB_DISTRIBUTOR="TrueNAS Scale"',
-        'GRUB_CMDLINE_LINUX_DEFAULT=""',
+        'GRUB_CMDLINE_LINUX_DEFAULT="libata.allow_tpm=1"',
```

Comment:
This is also possible to control via a /proc or /sys (dont remember which). Wonder if its better to keep it in t he plugin to keep it self-contained?